pkg/api/metrics.go

```diff
@@ -1,10 +1,114 @@
 package api
 
 import (
+	"time"
+
 	"github.com/prometheus/client_golang/prometheus"
+	"github.com/timescale/timescale-prometheus/pkg/pgclient"
 	"github.com/timescale/timescale-prometheus/pkg/util"
 )
 
+const (
+	tickInterval = time.Second
+)
+
+var (
+	metrics *Metrics
+
+	leaderGauge = prometheus.NewGauge(
```

Comment:
nit and optional refactor : do we need these vars? can't we just init the struct fields in `InitMetrics` and then register the fields? i.e.

```
InitMetrics() {
...
metrics = &Metrics{
          receivedSamples: prometheus.NewCounter(
		prometheus.CounterOpts{
			Namespace: util.PromNamespace,
			Name:      "received_samples_total",
			Help:      "Total number of received samples.",
		},
	)
...
prometheus.MustRegister(metrics.receivedSamples)
...
}
```

Reply:
Good catch!